opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -46,80 +47,55 @@ namespace Opm {
 
     namespace GridPropertyPostProcessor {
 
-        class DistributeTopLayer : public GridPropertyBasePostProcessor<double>
-        {
-        public:
-            DistributeTopLayer(const EclipseState& eclipseState) :
-                m_eclipseState( eclipseState )
-            { }
+        std::vector< double >& distTopLayer( std::vector<double>& values, const Deck& m_deck, const EclipseState& m_eclipseState ) {
```

Comment:
Is it necessary to return the values? As long as this function indeed mutates the input argument `values` I would prefer the function to be `void` - then everyone and his brother will understand that mutation of input argument take place.